ci/scripts/build-chip-verilator.sh

```diff
@@ -22,14 +22,15 @@ case "$tl" in
         fusesoc_core=lowrisc:dv:chip_verilator_sim
         vname=Vchip_sim_tb
         verilator_options="--threads 4"
+        make_options="-j"
```

Comment:
For CI we probably also have to match the number of threads to the instances we actually have, see my comment below.

Reply:
the earl grey verilator build is controleed in another file, AFAIK, and runs on a single threaded model I've set to build with -j 1. But -j4 is reasonable here.